src/test/java/Homework19.java

```diff
@@ -0,0 +1,51 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+
+        String deleteNotification = "Deleted playlist lazy days";
+
+        enterEmail("christina.taylor@testpro.io");
+
+        enterPassword("jKV0uSX6z1dv");
+        submit();
+        //select lazy days
+        clickLazyDays();
+        // delete button
+
+        removePlaylist();
+        // ok button to conform
+        //Verify that the confirmation notification
+        // displayed has the text, "Deleted playlist {playlist name}".
+
+        // notification message
+
```

Comment:
**Use in test methods, the following logical split   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**